### ISSUE TITLE
Can you treat an annotation as a window?

### ISSUE THREAD

**Issue by mejackreed** [ProjectMirador/mirador-design#28]
Brought up in Mirador session, can you promote an annotation as a first class window.

**Comment by ggeisler** [comment]
You can open the annotations sidebar panel as a companion window, as shown in this page of the mockups: https://xd.adobe.com/view/f2440f29-045b-4d8d-7b61-0d5c8ece8104-8475/screen/a9d42383-d70a-4dc8-a4b7-a165184ccf43/Companion-window-vertical

But that isn't a _single annotation_. Is that what the question means? Opening a single annotation in a new window outside of the sidebar?

**Comment by dfdan** [comment]
Yes - (this was a suggestion I made in today's session) - that an annotation should be promotable to its own slot (in Mirador 2 terms) - ie the current workspace would divide in half and the annotation would appear in a new slot (through a dynamically generated canvas.

With the proposed Mirador 3 designs this would be very powerful - eg collecting clippings and grouping on a common theme by dragging them around.

**Comment by ggeisler** [comment]
Okay, great, thanks for clarifying @dfdan. I'll discuss with @jvine to figure out the implications, etc.

**Comment by aeschylus** [comment]
Sounds like liquidtext (but for images): https://www.liquidtext.net/
